deepchem/feat/molecule_featurizers/raw_featurizer.py

```diff
@@ -45,6 +45,6 @@ def _featurize(self, mol: RDKitMol) -> Union[str, RDKitMol]:
       raise ImportError("This class requires RDKit to be installed.")
 
     if self.smiles:
-      return Chem.MolToSmiles(mol)
+      return [Chem.MolToSmiles(mol)]
```

Comment:
This looks to shift the return type here which I think is causing issues

Reply:
Putting `pdb.set_trace` at that line and I do not see pytest stop there, seems like it have nothing to do with the failed test